### PR TITLE
DHFPROD-6049: Confusion because operations don't apply to custom steps

### PR DESCRIPTION
**Pull request by AkshayBajajML**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

**Comment by AkshayBajajML** [comment]
UI changes reviewed by UX team.

**Review by briantang — APPROVED** [review]
Looks good, tests pass. https://drive.google.com/file/d/1TaGu6JEIXgGgALerm46u4fZzrcZHD1TF/view

File: marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx [hunk 1]
```diff
@@ -142,13 +142,15 @@ describe("Entity Tiles component", () => {
       await fireEvent.click(customerPanel);
       // Check for Mapping tab
       expect(getByText('Map')).toBeInTheDocument();
-      expect(getByText('Custom')).toBeInTheDocument();
+      await fireEvent.click(getByText('Custom'));
+      expect(getByText("You can create Custom steps either manually or using Gradle, then deploy them. Deployed Custom steps appear here. Hub Central only allows running Custom steps, not editing or deleting them.")).toBeInTheDocument();
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx:146]: I prefer not explicitly testing long strings of text and instead labeling those passages with an aria-label and testing for that. That way we can make minor changes to the text content without having to worry about the tests. I recommend adding:

aria-label={'customEntityTitle'}

and then check like so:

expect(getByLabelText('customEntityTitle')).toBeInTheDocument();

File: marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx [hunk 2]
```diff
@@ -142,13 +142,15 @@ describe("Entity Tiles component", () => {
...
       // Check for Matching tab
       expect(queryAllByText('Match')).toHaveLength(0);
       await fireEvent.click(customerPanel);
 
       await fireEvent.click(NoEntityTypePanel);
       //Custom card 'customXML' should be selected by default
       expect(getByText('customXML')).toBeInTheDocument();
+      expect(getByText('Steps that are created outside Hub Central and are not associated with any entity type appear here. Hub Central only allows running these steps, not editing or deleting them.')).toBeInTheDocument()
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx:154]: Similar to the above but:

aria-label={'customNoEntityTitle'}

and:

expect(getByLabelText('customNoEntityTitle')).toBeInTheDocument();

**Review by wooldridge — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx (hunk 1 above)
Comment (AkshayBajajML) [reply marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx]: Updated!!

File: marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx (hunk 2 above)
Comment (AkshayBajajML) [reply marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx]: Updated!!

**Review by wooldridge — APPROVED** [review]

**Review by briantang — APPROVED** [review]

**Review by timur-isangulov — APPROVED** [review]

**Pull request merged**